File: Survey/Assets/Models/Question.cs

using System.Collections.Generic;
[System.Serializable]
public class Question 
{
   public string _id;
   public string Label;
   public string QuestionTxt;
   public string Creator;
   public string Visibility;
   
   public List<Options> OptionsList = new List<Options>();
}

File: server/Models/Question.js
const mongoose = require('mongoose');
const fileSchema = new mongoose.Schema(
    {
        Label:String,
        QuestionTxt :String,
        OptionsList: Array,
        Creator: String,
        Visibility:String,
        
      
},
 {
    collection: 'Question'
 });

module.exports = mongoose.model('Question', fileSchema);